src/functionalTest/kotlin/com/vaadin/gradle/MiscMultiModuleTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.vaadin.gradle
+
+import org.junit.Test
+
+class MiscMultiModuleTest : AbstractGradleTest() {
+    /**
+     * Tests https://github.com/vaadin/vaadin-gradle-plugin/issues/38
+     */
+    @Test
+    fun testFileNotFoundException() {
```

Comment:
So we test that web waits until lib is built and doesn't fail on missing file. 
I feel the test name could be more descriptive as now it makes one expect that the test should fail on an exception.

Reply:
Good catch, thanks!